core/src/main/scala/kafka/controller/TopicDeletionManager.scala

```diff
@@ -156,7 +128,7 @@ class TopicDeletionManager(controller: KafkaController,
       val topicsToResumeDeletion = topics & topicsToBeDeleted
       if(topicsToResumeDeletion.nonEmpty) {
         topicsIneligibleForDeletion --= topicsToResumeDeletion
-        resumeTopicDeletionThread()
+        resumeDeletions()
```

Comment:
This can be done in a followup patch, but it doesn't seem that resumeDeletionForTopics() and resumeDeletionForTopics() need to be called due to preferred leader election.